src/Native/Runtime/thread.cpp

```diff
@@ -1287,4 +1312,16 @@ COOP_PINVOKE_HELPER(void, RhpReversePInvokeReturn2, (ReversePInvokeFrame * pFram
     pFrame->m_savedThread->InlineReversePInvokeReturn(pFrame);
 }
 
+COOP_PINVOKE_HELPER(void, RhpPInvoke2, (PInvokeTransitionFrame* pFrame))
```

Comment:
RhpPInvoke2/RhpPInvokeReturn2 should be under `#ifdef USE_PORTABLE_HELPERS` here. It does not make sense otherwise.